pyproject.toml

```
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools>=67.7.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+# please adapt to package name
+name = "ghga_datasteward_kit"
+version = "1.0.0"
+description = "GHGA Data Steward Kit - A utils package for GHGA data stewards."
+readme = "README.md"
+authors = [
+    { name = "German Human Genome Phenome Archive (GHGA)", email = "contact@ghga.de" },
+]
+requires-python = ">=3.9"
+license = { text = "Apache 2.0" }
+classifiers = [
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.9",
+    "License :: OSI Approved :: Apache Software License",
+    "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+dependencies = [
+    "crypt4gh==1.6",
+    "hexkit[s3]>=0.10,<0.11",
```

Isn't it compatible with the latest version?
```suggestion
    "hexkit[s3]~=0.11",
```